### ISSUE TITLE
Exception while register event with timestamp because of types

### ISSUE THREAD

**Issue by maximmarakov** [PostHog/posthog-python#5]
Capture get str object
https://github.com/PostHog/posthog-python/blob/master/posthog/__init__.py#L25
But 
_enqueue raise exception, because expect datetime.datetime object
https://github.com/PostHog/posthog-python/blob/master/posthog/client.py#L206
Then i catch exception : 
```
  File "/usr/local/lib/python3.7/site-packages/posthog/__init__.py", line 45, in capture
    _proxy('capture', distinct_id=distinct_id, event=event, properties=properties, context=context, timestamp=timestamp, message_id=message_id)
  File "/usr/local/lib/python3.7/site-packages/posthog/__init__.py", line 141, in _proxy
    fn(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/posthog/client.py", line 108, in capture
    return self._enqueue(msg)
  File "/usr/local/lib/python3.7/site-packages/posthog/client.py", line 214, in _enqueue
    require('timestamp', timestamp, datetime)
  File "/usr/local/lib/python3.7/site-packages/posthog/client.py", line 280, in require
    raise AssertionError(msg)
AssertionError: timestamp must have <class 'datetime.datetime'>, got: 2020-04-28 09:39:49
```

**Comment by timgl** [comment]
Yep good catch, the types are wrong. I think it does still make sense to require a datetime, as I don't want to get into the world of trying to parse random datestrings.

**Comment by timgl** [comment]
Fixed! Should be version 1.0.11
